src/main/java/com/adamk33n3r/runelite/watchdog/EventHandler.java

```diff
@@ -209,9 +209,14 @@ private void onItemContainerChanged(ItemContainerChanged itemContainerChanged) {
             return;
         Item[] items = itemContainerChanged.getItemContainer().getItems();
         long itemCount = Arrays.stream(items).filter(item -> item.getId() > -1).count();
+        Map<Integer, Integer> currentItems = new HashMap<>();
         Map<Integer, Integer> allItems = new HashMap<>();
```

Comment:
So the difference is now `currentItems` is what `allItems` used to be, and now `allItems` is `currentItems + previousItems(if missing)`? I think that's making sense to me.

Reply:
Yup! Exactly right. I figured the rename was necessary since it wasn't really correct haha

Although a technicality, `allItems` is technically `currentItems + (names of previous items)`. Since it doesn't actually add any quantities from the previous items list, just the existence of them.

Reply:
Right, set to `0`. 👍🏻